src/device.js

```diff
@@ -77,7 +77,31 @@ var devicePixelRatio = window.originalDevicePixelRatio || window.devicePixelRati
 // @deprecated
 exports.devicePixelRatio = devicePixelRatio;
 
-exports.screen.devicePixelRatio = devicePixelRatio;
+var ctx = document.createElement('Canvas').getContext('2d');
+
+// Provide a pixel ratio for our canvas
+Object.defineProperty(exports.screen, 'devicePixelRatio', {
```

Comment:
- either make this computed once or provide a comment explaining why it needs to be computed every time it is accessed
 - probably shouldn't export something called `devicePixelRatio` if it's not the same as the html5 `devicePixelRatio` (unless it's actually a bug fix for invalid `devicePixelRatio` values?) -- could just export `screen.devicePixelRatio` and `screen.backingStoreRatio`